common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -880,7 +883,14 @@ def get_client_key_secret(self):
         course = self.get_course()
         for lti_passport in course.lti_passports:
             try:
-                lti_id, key, secret = [i.strip() for i in lti_passport.split(':')]
+                # Start Changes by Labster: To simplify license applying UX
+                values = [i.strip() for i in lti_passport.split(':')]
+                if len(values) === 4:  # new Labster passport LTI format
```

Comment:
Why this using '===', I think it will be valid on javascript?
does it make sense if we just using '==' instead of 'a == b and type(a) == type(b)[python code]' ?